core/frontend/src/ViewContext.ts

```diff
@@ -1,462 +1,462 @@
-/*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-/** @packageDocumentation
- * @module Rendering
- */
-
-import { assert, Id64String } from "@itwin/core-bentley";
```

Comment:
Looks like this file was swapped from LF -> CRLF line endings. Can you switch it back to LF?